### PR TITLE
Added proposition-event schema to the decisioning layer of the experi…

### PR DESCRIPTION
**Pull request by shagreel**
Added proposition-event schema to the decision layer of the experienceevent-proposition-interaction schema

### CODE REVIEW

File: extensions/adobe/experience/decisioning/experienceevent-proposition-interaction.schema.json [hunk 1]
```diff
@@ -17,6 +17,11 @@
     "decisioning-experienceevent-all": {
       "type": "object",
       "properties": {
+        "https://ns.adobe.com/experience/decisioning/propositionEventType": {
+          "title": "Associated Event",
+          "description": "A record of how the user interacted with the proposition. All proposition recorded on this event will be associated with the same event that is recorded here.",
```
Comment (donwalling) [inline extensions/adobe/experience/decisioning/experienceevent-proposition-interaction.schema.json:22]: All proposition**s** ...

File: extensions/adobe/experience/decisioning/experienceevent-proposition-interaction.schema.json [hunk 2]
```diff
@@ -17,6 +17,11 @@
     "decisioning-experienceevent-all": {
       "type": "object",
       "properties": {
+        "https://ns.adobe.com/experience/decisioning/propositionEventType": {
+          "title": "Associated Event",
```
Comment (donwalling) [inline extensions/adobe/experience/decisioning/experienceevent-proposition-interaction.schema.json:21]: I'd like to think of some other word than "Associated" here - perhaps something that distinguishes this is a system  type whereas the other eventType is a cusotmer-defined type.
That said, I don't have a better idea ATM

**Review by donwalling — APPROVED** [review]

File: extensions/adobe/experience/decisioning/proposition-event-type.schema.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  "meta:status": "stable",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "definitions": {
+    "proposition-event": {
+      "properties": {
+        "https://ns.adobe.com/experience/decisioning/proposition-event/send": {
+          "title": "send",
+          "description": "Proposition was sent to the user. This does not guarantee the proposition arrived.",
```
Comment (ewiener) [inline extensions/adobe/experience/decisioning/proposition-event-type.schema.json:22]:
```suggestion
          "description": "Proposition was sent to the client. This does not guarantee the proposition arrived or was displayed to the user.",
```

**Comment by prabhum2** [comment]
@shagreel Please let me know when this PR is ready for XDM team's review and release into registry service

File: extensions/adobe/experience/decisioning/experienceevent-proposition-interaction.schema.json (hunk 2 above)
Comment (shagreel) [reply extensions/adobe/experience/decisioning/experienceevent-proposition-interaction.schema.json:21]: Would it be better to just call it "Proposition Event"?

**Comment by shagreel** [comment]
> @shagreel Please let me know when this PR is ready for XDM team's review and release into registry service

@prabhum2 This PR is ready

File: extensions/adobe/experience/decisioning/proposition-event-type.schema.json [hunk 4]
```diff
@@ -0,0 +1,48 @@
+{
+  "meta:license": [
+    "Copyright 2022 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/decisioning/proposition-event-type",
```
Comment (mjasrotia) [inline extensions/adobe/experience/decisioning/proposition-event-type.schema.json:8]: Are you defining a data type here ? Looks like yes as I see that you have referenced this in the "propositionEventType" property in the [experienceevent-proposition-interaction.schema.json] field group.

If this is a data type, please remove meta:intendedToExtend as that is only to define field group and its association to a class.

File: extensions/adobe/experience/decisioning/proposition-event-type.schema.json [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "AJO Classification Fields",
+  "description": "A set of direct customer interactions that an end-user might perform when shown a proposition.",
+  "type": "object",
+  "meta:status": "stable",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
```
Comment (mjasrotia) [inline extensions/adobe/experience/decisioning/proposition-event-type.schema.json:16]: Remove this if this is a data type

File: extensions/adobe/experience/decisioning/proposition-event-type.schema.json (hunk 4 above)
Comment (shagreel) [reply extensions/adobe/experience/decisioning/proposition-event-type.schema.json:8]: Yes. I have removed the metadata field.

**Review by prabhum2 — APPROVED** [review]

**Pull request merged**